lib/combinat.gi

```diff
@@ -182,14 +182,13 @@ CombinationsK := function ( mset, m, n, k, comb, i )
 end;
 MakeReadOnlyGlobal( "CombinationsK" );
 
-InstallGlobalFunction(Combinations,function ( arg )
-    local   combs, mset;
-    if Length(arg) = 1  then
-        mset := ShallowCopy(arg[1]);  Sort( mset );
+InstallGlobalFunction(Combinations,function ( mset, arg... )
```

Comment:
What is the advantage of splitting mset from arg?

Reply:
Ah, I see.
Is it faster to work with arg and to ask about the number of arguments in the GAP function or to write several GAP functions with different numbers of arguments and let GAP choose the right one?
Of course, it is more complex to maintain the same code in several functions.

Reply:
To let GAP "choose" the "right" function would mean to use an operation (via `DeclareOperation`), not a plain global function. And that incurs more overhead than a simple global function. Besides, it would be a much bigger change than the simple refactoring done in this PR.